style.css

```diff
@@ -0,0 +1,472 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
```

Comment:
use indicator for individual section start and end @SuhaniKhatri

Reply:
Means I should add comments from where styling of first section started and ended?